packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -686,7 +687,7 @@ export class Repository extends Construct implements IRepository {
       const instances = props.documentDbInstanceCount ?? Repository.DEFAULT_NUM_DOCDB_INSTANCES;
       const dbCluster = new DatabaseCluster(this, 'DocumentDatabase', {
         masterUser: {username: 'DocDBUser'},
-        engineVersion: '3.6.0',
+        engineVersion: '5.0.0',
```

Comment:
Looks like we're still using DocumentDB 3.6 in a few places such as `examples/deadline/All-In-AWS-Infrastructure-Basic/ts/lib/storage-tier.ts`.  Do we need to update those too?

Reply:
I'll update this

Reply:
Actually, I'll just update the example, since I don't want to update the integ structs without doing a retest, and I don't think we have bandwidth for a retest

Reply:
Updated this